index.bs

```diff
@@ -10306,11 +10250,54 @@ of [=inherited interfaces=] for any
 other interface that is declared with one of these attributes, then the [=interface prototype object=]
 must be an [=immutable prototype exotic object=].
 
-The [=class string=] of an
-[=interface prototype object=]
-is the concatenation of the [=interface=]’s
-[=identifier=] and the string
-“Prototype”.
+The [=class string=] of an [=interface prototype object=] is the concatenation of
+the [=interface=]’s [=identifier=] and the string “Prototype”.
+
+
+<h4 id="legacy-callback-interface-object">Legacy callback interface object</h4>
+
+For every callback [=interface=] that is [=exposed=] in
+a given ECMAScript global environment
+and on which [=constants=] are defined,
+a corresponding property must exist on the ECMAScript environment's global object.
+The name of the property is the [=identifier=] of the interface,
+and its value is an object called the
+<dfn id="dfn-legacy-callback-interface-object" export>legacy callback interface object</dfn>.
+The property has the attributes { \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val> }.
+The characteristics of a legacy callback interface object are described in [[#legacy-callback-interface-object]].
```

Comment:
This last sentence is just a link to the section that the sentence is already in, which seems bad.